### ISSUE TITLE
Error when run e2e-tests

### ISSUE THREAD

**Issue by krivochenko** [nestjs/graphql#58]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Error when run e2e-tests:
```
TypeError: this.httpServer.getInstance is not a function

      at GraphQLModule.<anonymous> (node_modules/@nestjs/graphql/dist/graphql.module.js:91:41)
      at node_modules/@nestjs/graphql/dist/graphql.module.js:19:71
      at Object.<anonymous>.__awaiter (node_modules/@nestjs/graphql/dist/graphql.module.js:15:12)
      at GraphQLModule.onModuleInit (node_modules/@nestjs/graphql/dist/graphql.module.js:89:16)
      at NestApplication.callModuleInitHook (node_modules/@nestjs/core/nest-application-context.js:66:39)
```

## Expected behavior

No error

## Minimal reproduction of the problem with instructions

1. git clone git@github.com:krivochenko/graphql-testing.git
2. npm i
3. npm run start (just for ensure that app works)
4. npm run test

## Environment

<pre><code>
Nest version: 5.3.2

 
For Tooling issues:
- Node version: 10.1.0
- Platform:  Windows
</code></pre>

**Comment by agborkowski** [comment]
youp same for me
Env:
node -v v10.9.0
osx
npm -v 6.2.0

**Comment by cschroeter** [comment]
I can copy that. If you add the GraphQL Module 
```typescript
GraphQLModule.forRoot({
      typePaths: ['./**/*.graphql']
 })
``` 
the e2e tests will fail.

**Comment by cyril-lakech** [comment]
exact same situation here

**Comment by cyril-lakech** [comment]
can be reproduced here https://github.com/cyril-lakech/nest/tree/test/e2eGraphQL/sample/12-graphql-apollo

**Comment by kamilmysliwiec** [comment]
Fixed in 5.2.0. Ensure to update Nest packages (core, common) as well.

**Comment by cyril-lakech** [comment]
Thanks you so much
